clusterloader2/testing/list/config.yaml

```diff
@@ -0,0 +1,189 @@
+# List benchmark
+{{$namespaceNumber := DefaultParam .CL2_LIST_NAMESPACE_NUMBER 4}}
+{{$configMapBytes := DefaultParam .CL2_LIST_CONFIG_MAP_BYTES 10000}}
+{{$configMapGroup := DefaultParam .CL2_LIST_CONFIG_MAP_GROUP "list-configmap"}}
+{{$configMapNumber := DefaultParam .CL2_LIST_CONFIG_MAP_NUMBER 3000}}
+{{$secretBytes := DefaultParam .CL2_LIST_SECRET_BYTES 10000}}
+{{$secretGroup := DefaultParam .CL2_LIST_SECRET_GROUP "list-secret"}}
+{{$secretNumber := DefaultParam .CL2_LIST_SECRET_NUMBER 3000}}
+
+{{$listReplicas := DefaultParam .CL2_LIST_BENCHMARK_PODS 1}}
+{{$inflight := DefaultParam .CL2_LIST_CONCURRENCY 2}}
+{{$qps := DefaultParam .CL2_LIST_QPS -1}}
+
+{{$namespacedPodCpu := DefaultParam .CL2_LIST_BENCHMARK_POD_CPU "400m"}}
+{{$namespacedPodMemory := DefaultParam .CL2_LIST_BENCHMARK_POD_MEMORY "800Mi"}}
+{{$clusterScopedPodCpu := DefaultParam .CL2_LIST_BENCHMARK_POD_CPU "2000m"}}
+{{$clusterScopedPodMemory := DefaultParam .CL2_LIST_BENCHMARK_POD_MEMORY "4000Mi"}}
```

Comment:
Any opportunity to get rid of some of these variables either by using some sane defaults or auto-calculating based on the load parameters?

More options usually mean more confusion and harder to adopt by other devs :)

Reply:
Done